src/entryconfig.ts

```diff
@@ -91,6 +98,11 @@ export async function loadEntryConfig(
       mod.inputs = mod.inputs.map(input => path.resolve(basedir, input));
     });
   }
+  if (entryConfig.warningsWhitelist) {
+    Object.values(entryConfig.warningsWhitelist).forEach(item => {
```

Comment:
Why is this `Object.values` required? Is there any problem just to use `entryConfig.warningsWhitelist.forEach`?

Reply:
@koba04 nice catch!

Reply:
@koba04 fixed! https://github.com/teppeis/duck/commit/1f05ed90527148037c2123a61b36500ab5e85555

Reply:
Thanks!